test/RoundTrip/Spec.hs

```diff
@@ -2,19 +2,15 @@
 {-# LANGUAGE DataKinds #-}
 {-# LANGUAGE OverloadedStrings #-}
 {-# LANGUAGE TypeApplications #-}
+{-# OPTIONS_GHC -Wno-missing-import-lists #-}
```

Comment:
If you want to leave this maybe have a `FIXME`

Reply:
we're doing another round of cosmetics as part of https://github.com/mlabs-haskell/purescript-bridge/issues/5